cmd/lib/runfile/cmd.go

```diff
@@ -279,6 +285,29 @@ func setUpBQLStmt(tb *bql.TopologyBuilder, bqlFile string) error {
 	return nil
 }
 
+func checkStates(tb *bql.TopologyBuilder, saveUDSList string) error {
+	if saveUDSList == "" {
+		return nil // save all UDS and pass to check
+	}
+	states, err := tb.Topology().Context().SharedStates.List()
+	if err != nil {
+		return err
+	}
+	notFoundFlag := false
```

Comment:
It's obvious from its type (bool) that this variable is a flag, so `Flag` isn't necessary.